docker/python-base/environment.yml

```diff
@@ -3,20 +3,20 @@ name: base
 #
 #  Dependencies include a small set of build-time
 #  dependencies to ease compiling libraries.
+channels:
+  - conda-forge
 dependencies:
-  - blas=1.0
+  - libblas=*=*mkl
```

Comment:
what does  `=*=*` mean? can you add a comment

Reply:
done